test/sql/functap.sql

```diff
@@ -1,8 +1,8 @@
 \unset ECHO
 \i test/setup.sql
 
-SELECT plan(640);
---SELECT * FROM no_plan();
+SELECT plan(1012);
```

Comment:
I wonder at what point test clarity is improved by meta-programming for the tests... (ie, create rowsets like "is_normal_function, isnt_normal_function", "'{}', array[etype()]", etc and then cartesian them together in a SELECT)

Reply:
I've done stuff like that, just written queries to test a whole bunch of things in a single expression. Is that what you mean?

Reply:
That, or there's other ways to do metaprogramming too. Main point is that 600 tests seems like a really big maintenance burden, let alone 1200.

Reply:
That's fair. Part of it is how it has grown over time. Probably would be useful to break that file up, similar to how I put the new procedure testing functions in a separate file. Do it for window and aggregate tests, too.